.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: PyTables Build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  pytables-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
+    steps:
+      - name: 'Check out repo content'
+        uses: actions/checkout@v2
+      - name: 'Set up Python ${{ matrix.python-version }}'
+        uses: actions/setup-python@v2
+        with:
+          python-version: '${{ matrix.python-version }}'
+      # - run: sudo apt-get install libbz2-dev libhdf5-serial-dev liblzo2-dev
+      - name: 'Run test'
+        run: |
+          pip install cython
+          pip install dist/*.tar.gz
```

Comment:
IMHO before running
```
pip install dist/*.tar.gz
```
there should be something like
```
python3 setup.py sdist
```
or
```
make dist
```

Reply:
The long run is still now over. I will now change that line to `python3 setup.py dist` and limit Python versions to 3.5.